app/controllers/SessionsController.scala

```diff
@@ -45,7 +47,13 @@ case class KnolxSession(id: String,
                         meetup: Boolean,
                         cancelled: Boolean,
                         rating: String,
-                        feedbackFormScheduled: Boolean = false)
+                        feedbackFormScheduled: Boolean = false,
+                        dateString: String = "",
+                        completed: Boolean = false)
+
+case class SessionEmailInformation(email: Option[String], page: Int)
+
+case class SessionSearchResult(sessions: List[KnolxSession], pages: Int, page: Int, keyword: String)

```

Comment:
Can we use the following format for class, methods with parameters > 3

> case class SessionSearchResult(sessions: List[KnolxSession],
                                                        pages: Int,
                                                        page: Int,
                                                        keyword: String)